content/en/account_management/rbac/permissions.md

```diff
@@ -319,7 +347,7 @@ Grants a role read access on some number of log indexes. Can be set either globa
 
```

Comment:
Does this no longer work if the `logs_read_data` is not applied? We have customers (Disney+, Starbucks) that have automated sync to keep their RBAC in sync with their AWS roles and permissions. Introducing a new permission will be overwritten, so we need to clarify here the regression if `logs_read_data` is now required that this permission is now `restrictive`